pySDC/tests/test_projects/test_resilience/test_rk.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+from pySDC.playgrounds.Runge_Kutta.test_order import test_vdp, test_advection
+
```

Comment:
So, I'd rather avoid importing from the playgrounds.. it's wild west out there. Rather put the files into your project, make a new project or add the test directly to the `test` directory (where stuff like the IMEX sweeper are tested, too).